lib/units/ios-device/plugins/wda.js

```diff
@@ -26,6 +26,8 @@ module.exports = syrup.serial()
             iosutil.pressButton.call(wdaClient, message.key)
           })
         }
+        // On portait mode -> home button directly
+        iosutil.pressButton.call(wdaClient, message.key)
```

Comment:
is it home button click for every new session?

Reply:
No, just processing the home button request:
1) Portrait mode -> click home button -> send `homeBtn` request 
2) Landscape mode -> click home button -> rotate to portrait mode -> send `homeBtn` request